lol-frontend/src/config/store/player/types.ts

```diff
@@ -0,0 +1,9 @@
+import { editPlayerPosition } from "./actions";
+
+export interface PlayerState {
+  readonly position: number[];
+}
+
+// Action Types
+
+export type PlayerActionTypes = ReturnType<typeof editPlayerPosition>;
```

Comment:
if this is supposed to be the model for your PlayerState then you do not need to import the actions or export them the interface is all you needed.

Reply:
If you were trying to define the type for the action then it is a little more in depth.

This would be where you would define the shape of your action. You would start by declaring variables like we did in regular old redux:
export const SET_PLAYER_POSITION = 'SET_PLAYER_POSITION';

Then you will want to define the actions model:
interface SetPlayerPositionAction {
    type: typeof SET_PLAYER_POSITION;
    payload: number[]
}

Then you will export the Action's types, and you will seperate each model with a pipe ( | ) symbol:
export type PlayerActionTypes = 
    SetPlayerPositionAction | BlahBlahAction